serving/docker/deepspeed.Dockerfile

```diff
@@ -14,7 +14,7 @@ FROM nvidia/cuda:$version
 ARG djl_version=0.20.0~SNAPSHOT
 ARG torch_version=1.12.1
 ARG accelerate_version=0.13.2
-ARG deepspeed_version=0.7.5
+ARG deepspeed_wheel="https://djl-misc.s3.amazonaws.com/deepspeed/deepspeed-0.7.5%2Bbf16-py2.py3-none-any.whl"
```

Comment:
```suggestion
ARG deepspeed_wheel="https://publish.djl.ai/deepspeed/deepspeed-0.7.5%2Bbf16-py2.py3-none-any.whl"
```

Reply:
are you suggesting to make the artifacts in prod bucket?